src/main/java/com/sonyericsson/jenkins/plugins/bfa/statistics/StatisticsLogger.java

```diff
@@ -126,7 +126,7 @@ public void run() {
             Node node = build.getBuiltOn();
             String nodeName = node.getNodeName();
             int timeZoneOffset = TimeZone.getDefault().getRawOffset();
-            String master = "";
+            String master;
```

Comment:
Initializing with empty string is useless here because in line 140 it will be initialized.
`master = BfaUtils.getMasterName();`